prebuild_binaries/native/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM debian:jessie
+
+RUN apt-get update
+RUN apt-get install -y curl build-essential python libc6-dev-i386 lib32stdc++-4.9-dev
+RUN curl -fsSL get.docker.com | bash
```

Comment:
What does this do?

Reply:
This installed docker on the docker image. 
It's not necessary here, so I removed it.